sopt/src/main/scala/dagr/sopt/cmdline/CommandLineProgramParser.scala

```diff
@@ -280,15 +280,16 @@ class CommandLineProgramParser[T](val targetClass: Class[T]) extends CommandLine
   }
 
   /** Gets the command line assuming `parseTasks` has been called */
-  def commandLine(printCommon: Boolean = true): String = {
+  def commandLine(): String = {
     val argumentList = this.argumentLookup.ordered.filterNot(_.hidden)
     val toolName: String = targetName
     val commandLineString: StringBuilder = new StringBuilder
 
     commandLineString.append(argumentList.filter( _.hasValue).filter(!_.isSpecial).map(_.toCommandLineString).mkString(" "))
```

Comment:
I know you didn't change it, but would you mind fixing the `filter(!_.isSpecial)` to be `filterNot(_.isSpecial)` please?